packages/workbox-webpack-plugin/test/node/test.js

```diff
@@ -61,13 +61,13 @@ describe('Tests for webpack plugin', function() {
 
   it('should mutate config accordin to webpack defaults', () => {
     let swWebpackPlugin = new SwWebpackPlugin({});
-    assert.equal(swWebpackPlugin.getConfig(webpackCompilation).rootDirectory,
+    assert.equal(swWebpackPlugin.getConfig(webpackCompilation).globDirectory,
```

Comment:
How was this test passing before? I would have thought it would fail given how long rootDirectory has been changed. Are you testing the final service worker output?